src/plugins/tabs/components/tabs.vue

```diff
@@ -88,11 +92,16 @@ export default {
             this.composition.on('add', this.addItem, this);
             this.composition.load();
         }
+
+        document.addEventListener('dragstart', this.dragstart, this);
```

Comment:
document.addEventListener doesn't have same arguments as `on`, passing `this` actually sets a capturing listener instead of setting function context.

Also, vue methods are all bound to the instance so it's not necessary to pass context arguments when attaching them as listeners.   You can remove `this` from calls above too.